stories/data/geoChartComponentProps.ts

```diff
@@ -8,12 +8,14 @@ export const ATTRIBUTE_LOCATION_GEOCHART: VisualizationObject.IVisualizationAttr
 export const MEASURE_SIZE_GEOCHART: VisualizationObject.IMeasure = measure(
     "/gdc/md/storybook/obj/20",
 ).localIdentifier("size");
-export const MEASURE_COLOR_GEOCHART: VisualizationObject.IMeasure = measure(
-    "/gdc/md/storybook/obj/21",
-).localIdentifier("color");
+export const MEASURE_COLOR_GEOCHART: VisualizationObject.IMeasure = measure("/gdc/md/storybook/obj/21")
+    .localIdentifier("color")
+    .alias("Color Alias");
 export const ATTRIBUTE_SEGMENT_GEOCHART: VisualizationObject.IVisualizationAttribute = attribute(
     "/gdc/md/storybook/obj/23.df",
-).localIdentifier("segmentBy");
+)
+    .localIdentifier("segmentBy")
+    .alias("Category Alias ");
```

Comment:
Please remove the blank character here

Reply:
Prettier did one.

Reply:
We have to create new `const MEASURE_COLOR_GEOCHART_WITH_ALIAS = MEASURE_COLOR_GEOCHART.alias('...')`
So that it does not impact current data

Reply:
I tested and the data didn't impact other part.

Reply:
Yes, just for now it does.
As your way, it affects all the geo storybook with alias for Color and Segment which is not really necessary.
In the future, we could add storybook tooltip test with backstopJs custom script, and there are test cases for tooltip with/out alias.

Reply:
fixed